lib/openxml/docx/elements/level.rb

```diff
@@ -0,0 +1,43 @@
+module OpenXml
+  module Docx
+    module Elements
+      class Level < OpenXml::Docx::Element
+        include HasChildren, HasProperties
+        tag :lvl
+
+        with_namespace :w do
+          attribute :level, expects: :integer, displays_as: :ilvl
+          attribute :tplc
+          attribute :tentative
+        end
```

Comment:
I know you've said this is a work-in-progress, but just wanted to make a note that these two attributes probably need better names and expectations before this can be merged.

Reply:
I think it turns out those two attributes are "UI only" attributes and will probably just be removed entirely, unless there's a pressing need to support entirely opaque Word attributes.

Reply:
Yeah, it doesn't look as if they're required in the spec. Maybe you can just comment them out, and they can serve as placeholders should we decide to implement them down the road.